### PR TITLE
convert-all-units

### PR DESCRIPTION
**Pull request by linkfang**
https://github.com/fang0055/2020-04-17-personal-npm-package

### CODE REVIEW

**Comment by linkfang** [comment]
Submit

**Comment by linkfang** [comment]
Final submission

**Comment by linkfang** [comment]
> Submit

https://www.npmjs.com/package/convert-all-units

File: Readme.md [hunk 1]
```diff
@@ -1,76 +1,4 @@
...
-1. You get a LOT of time for code review comments. So you can win even if you feel your project isn't great, simply on the strength of your code review comments!
-1. Keep in mind, 33% of your score is based on aesthetic appeal!
-
-## Resources
-
-* [Creating and publishing unscoped public packages](https://docs.npmjs.com/creating-and-publishing-unscoped-public-packages)
-* [Video - Creating & publishing your own NPM modules](https://www.youtube.com/watch?v=rTsz09zRuTU)
-* [freeCodeCamp's Tutorial](https://www.freecodecamp.org/news/how-to-make-a-beautiful-tiny-npm-package-and-publish-it-2881d4307f78/)
+## Features
+1. Convert between grams and pounds, kilograms ande pounds.
```
Comment (amberyiyao) [inline Readme.md:4]: You can list all the methods that can use from your library. And give the instruction about how to install and usage example.

**Pull request closed without merging**